site/en/posts/iwd-2022-shree-bose-piper.md

```diff
@@ -0,0 +1,66 @@
+
+As a woman in science, this was not going to work for me. There was a problem. I then sat down to try the kit out myself. It was fascinating! I had been creating this thing for such a long time so thought I knew what the experience was. However, when I actually sat down and built it from scratch myself, within 15 minutes, I too felt like “All right, I built the thing, I did the thing, I’m done.” There was a moment when everything clicked in my mind and I realised: “Well, of course I don't want to build anymore. Why would I? There's no reason for me to build more. What is my motivation?”
+
+This is why the storyline in Piper exists. At that moment, sitting in front of my own computer I had just assembled, I felt like I had no reason to continue building. So we thought about ways to change this experience and created that “reason” by adding in storylines. We took this new version into classrooms and, suddenly, an equal proportion of girls and boys we engaged. We did studies where, with this version, girls’ confidence playing with Piper goes up significantly more than for guys. Storytelling is a huge part of what makes Piper what it is today.
+
+The reason I bring it up is that when I think about what I would tell a brilliant young woman who might be reading this, it would be: sometimes you might feel yourself really, really struggling with something. You might have failed a physics test, you might have failed a chemistry test, you might feel like you are really, really bad at something. It's important to realize that sometimes the reason you feel like you suck at it is because it is not designed for you. It's really not designed to fit the way you learn. And that does not mean that you suck at it, at all. In fact, it might mean that you are absolutely brilliant at it, but you are not being taught that concept or that thing in the way that you are most able to pick it up. Don't give up on things too easily. We are still operating with educational models and paradigms that are not designed for women.
+
+> As you are in that moment, feeling that you are sucking at something, think about different ways to teach yourself the same thing because if you don't give up, you might come up with something really, really amazing. And that's the most powerful thing you can do.
+
+---
+
+I am so grateful to have had the opportunity to talk with Shree and I cannot wait to try out some of the upcoming kits myself. Check out [Piper and Piper Make](https://www.playpiper.in/) today with your Chromebook!
+
+Emilie Roberts, Developer Relations Engineer, Google Chrome OS
```

Comment:
```suggestion
```

You're already credited as the author through our template, so you don't need to include a by-line at the end.

Reply:
I am ok to not be credited here BUT I wonder if having doesn't make it clear that the remaining text is me and not Shree speaking?